community/cypher/cypher-compiler-2.3/src/test/scala/org/neo4j/cypher/internal/compiler/v2_3/planner/LogicalPlanningTestSupport.scala

```diff
@@ -169,7 +169,9 @@ trait LogicalPlanningTestSupport extends CypherTestSupport with AstConstructionT
       plannerName = None,
       runtimeBuilder = InterpretedRuntimeBuilder(InterpretedPlanBuilder(Clock.SYSTEM_CLOCK, monitors)),
       semanticChecker = semanticChecker,
-      useErrorsOverWarnings = false)
+      useErrorsOverWarnings = false,
+      idpMaxTableSize = 128,
+      idpIterationDuration = 1000)
```

Comment:
These aren't the IDP defaults from above. I think our general testing framework should use the defaults, and to accomplish this I suggest not hard-coding them here, but derive them from `DefaultIDPSolverConfig`.